lib/SPIRV/LLVMToSPIRVDbgTran.cpp

```diff
@@ -1365,17 +1365,23 @@ SPIRVEntry *LLVMToSPIRVDbgTran::transDbgExpression(const DIExpression *Expr) {
 SPIRVEntry *
 LLVMToSPIRVDbgTran::transDbgImportedEntry(const DIImportedEntity *IE) {
   using namespace SPIRVDebug::Operand::ImportedEntity;
-  SPIRVWordVec Ops(OperandCount);
   auto Tag = static_cast<dwarf::Tag>(IE->getTag());
+  // FIXME: 'OpenCL/bugged' version is kept because it's hard to remove it
+  // It's W/A for missing 2nd index in OpenCL's implementation
```

Comment:
Alternatively we could have 2 translation functions (see the first patch in this PR), but I ended up removing duplicated code and do the fix this way. Please communicate if you have different preferences.